index.js

```diff
@@ -59,6 +59,23 @@ class Machine {
     })
   }
 
+  static create(machineName, options, done) {
+    var args = [];
+    args.push('create');
+    for (var key in options) {
```

Comment:
I think I prefer to make the options explicit. E.g.:

```js
if (options.driver) args.push('--driver', options.driver)
```

Reply:
Ohh, you mean rather than looping through all the args one by one, you prefer to explicitly copy the args?
Well, for other drivers, say, for a "generic" driver, there are other additional arguments, for example, its command would be like

docker-machine create --driver generic --generic-ip-address=xxx.xx.xx.xx --generic-ssh-key ~/.ssh/id_rsa --generic-ssh-user root <machine-name>

The options vary for every single driver, so, I left that choice to the caller.